busses:
  - port: /dev/serial/by-id/usb-MIL_Data_Merge_Board_Ports_1_to_4_DMBP14-if00-port0
    thrusters:
      FLH: {
        node_id: 0,
        frame_id: /base_link,
        position: [0.2678, 0.2795, 0],
        direction: [-0.866, 0.5, 0]
      }
      FLV: {
        node_id: 1,
        frame_id: /base_link,
        position: [0.1583, 0.169, 0.0142],
        direction: [0, 0, -1]
      }
  - port: /dev/serial/by-id/usb-MIL_Data_Merge_Board_Ports_1_to_4_DMBP14-if01-port0
    thrusters:
      FRH: {
        node_id: 2,
        frame_id: /base_link,
        position: [0.2678, -0.2795, 0],
        direction: [0.866, 0.5, 0]
      }
      FRV: {
        node_id: 3,
        frame_id: /base_link,
        position: [0.1583, -0.169, 0.0142],
        direction: [0, 0, -1]
      }
  - port: /dev/serial/by-id/usb-MIL_Data_Merge_Board_Ports_1_to_4_DMBP14-if02-port0
    thrusters:
      BLH: {
        node_id: 4,
        frame_id: /base_link,
        position: [-0.2678, 0.2795, 0],
        direction: [0.866, 0.5, 0]
      }
      BLV: {
        node_id: 5,
        frame_id: /base_link,
        position: [-0.1583, 0.169, 0.0142],
        direction: [0, 0, 1]
      }
  - port: /dev/serial/by-id/usb-MIL_Data_Merge_Board_Ports_1_to_4_DMBP14-if03-port0
    thrusters:
      BRH: {
        node_id: 6,
        frame_id: /base_link,
        position: [-0.2678, -0.2795, 0],
        direction: [0.866, -0.5, 0]
      }
      BRV: {
        node_id: 7,
        frame_id: /base_link,
        position: [-0.1583, -0.169, 0.0142],
        direction: [0, 0, 1]

